packages/react-app-scaffolder/app/templates/_config.json

```diff
@@ -2,10 +2,9 @@
   "appEnv": "local",
   "sentryDns": "",
   "connectClientId": "<%= clientId %>",
-  "connectUserPoolId": "",
+  "connectUserPoolId": "<%= userPoolId %>",
   "googleAnalyticsKey": "",
-  "connectOAuthUrl": "",
-  "platformApiUrl": "",
-  "graphqlUri": "",
-  "marketplaceUrl": ""
+  "connectOAuthUrl": "https://connect.dev.paas.reapit.cloud",
+  "platformApiUrl": "https://platform.dev.paas.reapit.cloud",
+  "marketplaceUrl": "https://marketplace.dev.paas.reapit.cloud"
```

Comment:
Are there instructions on how to go live for this? Just double making sure

Reply:
Or is this for mono only?

Reply:
It's Mono Only